docs/tangle.md

```diff
@@ -0,0 +1,276 @@
+                    </tr>
+                    <tr>
+                        <td>Data Fields</td>
+                        <td>ANY</td>
+                        <td>A sequence of fields, where the structure depends on <code>payload type</code>.</td>
+                    </tr>
+                </table>
+            </details>
+            </td>
+    </tr>
+    <tr>
+        <td>Nonce</td>
+        <td>uint64</td>
+        <td>The nonce which lets this message fulfill the Proof-of-Work requirement.</td>
+    </tr>
+    <tr>
+        <td>Signature (Ed25519)</td>
+        <td>ByteArray[64]</td>
+        <td>Signature of the issuing node's private key signing the entire message bytes.</td>
+    </tr>
+</table>
+
+
+### Syntactical Validation
+1. The message length must not exceed `MAX_MESSAGE_SIZE` bytes.
+2. When we are done parsing the message there shouldn't be any trailing bytes left that were not parsed.
+4. At least 1 and maximum 8 distinct parents are given, ordered ASC and at least `MIN_STRONG_PARENTS` are strong parent. 
+
+### Semantic Validation / Eligibility
+Only eligible messages are processed by a node, added to its local Tangle and thus can become available as tips for tip selection. Eligibility of a message **does not** express any opinion about the attachment location of a message. It solely evaluates a message according to its timestamp.
+
+A message is an eligible message if:
+- If the Message PoW Hash will contain at least the number of leading 0 the node defines as required.
+-  Verify signature to be valid from issuing node.
+- It is solid
+- It has a good timestamp (TODO: link timestamp spec)
+- It passes parents age checks (TODO: link timestamp spec)
+- Its parents are eligible
+
+### Metadata
+Next to a message itself, a node needs to store additional data that describe its local perception of this message and are not part of the Tangle.
+
+<table>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>receivedTime</td>
+        <td>time</td>
+        <td>The local time the message was received by the node.</td>
+    </tr>
+    <tr>
+        <td>solid</td>
+        <td>bool</td>
+        <td>Denotes whether a message is solid, i.e., its past cone is known.</td>
+    </tr>
+    <tr>
+        <td>solidificationTime</td>
+        <td>time</td>
+        <td>The local time the message got solid.</td>
+    </tr>
+    <tr>
+        <td>opinion</td>
+        <td>Opinion</td>
+        <td>Contains a nodes' opinion on the timestamp of a message. A triple <code>(opinion, level, timeFormed)</code>, where opinion is a bool, level is in the set {1,2,3}, and timeFormed is a time. The opinionField is also manipulated by FPC. TODO: this should be defined probably in the FPC specs?</td>
+    </tr>
+    <tr>
+        <td>eligible</td>
+        <td>bool</td>
+        <td>Denotes whether a message is eligible, i.e., it is added to a node's local Tangle.</td>
```

Comment:
"Denotes whether a message is eligible, i.e., it is added to a node's local Tangle"

=>

"Denotes whether a message is eligible, i.e., it is available for TSA"